app/views/admin/ride_uploads/_form.html.haml

```diff
@@ -0,0 +1,23 @@
+= form_for( @ride_upload, url: admin_ride_zone_ride_uploads_path(@ride_zone)) do |f|
+  - if @ride_upload.errors.any?
+    #error_explanation
+      %h2= "#{pluralize(@ride_upload.errors.count, "error")} prohibited this ride_upload from being saved:"
+      %ul
+        - @ride_upload.errors.full_messages.each do |message|
+          %li= message
+
+  .col-sm-8.field
+    = f.label :name
```

Comment:
I wasn't sure when I first saw this in-app what "name" was for. Maybe some explanatory text would help? Assuming the below guess is right, 😄  maybe something like: 

```
    %i
      &mdash; for later reference, like "Precinct 8"
```

Reply:
Good call, done